torch/nested/_internal/ops.py

```diff
@@ -472,6 +472,31 @@ def prod_dim_int(func, *args, **kwargs):
     return NestedTensor(func(inp._values, **new_kwargs), **extract_kwargs(args[0]))
 
 
+@register_jagged_func(
+    torch.ops.aten.split.Tensor, "self: jt, split_size: any, dim: any"
+)
+def split_tensor(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    inp = new_kwargs.pop("input")
+    values = inp._values
+
+    # hack to split on the last dim
+    dim = new_kwargs["dim"]
+    if dim != -1:
+        raise RuntimeError(
+            "split(): only supported for NestedTensor on dim = -1 for now"
+        )
+
+    split_size = new_kwargs["split_size"]
+    return tuple(
+        NestedTensor(values=x, **extract_kwargs(inp))
+        for x in torch.split(values, split_size, -1)
```

Comment:
```suggestion
        for x in func(inp._values, **new_kwargs)
```
nit: no need to do `split_size = new_kwargs["split_size"]` etc, you can just use `new_kwargs` directly